provisioning_client/src/prov_transport_http_client.c

```diff
@@ -1231,7 +1231,15 @@ static int prov_transport_http_set_option(PROV_DEVICE_TRANSPORT_HANDLE handle, c
         }
         else
         {
-            result = uhttp_client_set_option(http_info->http_client, option, value);
+            HTTP_CLIENT_RESULT http_result = uhttp_client_set_option(http_info->http_client, option, value);
+            if (http_result != HTTP_CLIENT_OK)
+            {
+                result = MU_FAILURE;
```

Comment:
This looks good. Would it be worth having a LogError for the MU_FAILURE scenario?

Reply:
It didn't have it before so I'd like to leave as is. There are already a good amount of log messages that bubble up the stack